pynest/nest/lib/hl_api_helper.py

```diff
@@ -515,10 +514,9 @@ def model_deprecation_warning(model):
 
     if model in _deprecation_warning:
         if not _deprecation_warning[model]['deprecation_issued']:
-            text = "The {0} model is deprecated and will be removed in a \
-            future version of NEST, use {1} instead.\
-            ".format(model, _deprecation_warning[model]['replacement'])
-            text = get_wrapped_text(text)
+            text = \
+                "The {0} model is deprecated and will be removed in a future version of NEST, use {1} instead.".format(
+                    model, _deprecation_warning[model]['replacement'])
```

Comment:
I'm not sure that breaking the line with `\` here is the right way to go, I think the preferred way is to use parentheses.

Reply:
Parentheses can be avoided:
```Python
            text = "The {0} model is deprecated and will be removed in a future version of NEST, " \
                   "use {1} instead.".format(model, _deprecation_warning[model]['replacement'])
```

Reply:
We want to avoid the backslash. From [PEP8:](https://www.python.org/dev/peps/pep-0008)

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

And it looks like `.format()` works with implicitly continued strings.

Reply:
Personally I prefer the option that avoids brackets, but PEP8 is of course the standard for NEST Simulator, so I am dropping this point! Tabs versus spaces, however... :)